features/requestclose.yml.dist

```diff
@@ -2,11 +2,14 @@
 # Do not edit this file by hand. Edit the source file instead!
 
 status:
-  baseline: false
+  baseline: low
+  baseline_low_date: 2025-05-27
```

Comment:
This seems fine. A quick search revealed no open bugs for `requestClose()` on BCD or the browser bug trackers.